index.html

```diff
@@ -251,6 +264,8 @@ <h3 class="text-contact">
             </li>
           </ul>
         </form>
```

Comment:
- [ ] You have done an amazing job 👏 👏
- [ ] Please try again to stick as much as possible to the Figma design. 
- [ ] This picture below is from [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) 

![fig1](https://user-images.githubusercontent.com/63972670/160003749-7020eeba-0f50-485c-b003-a754ffb97929.JPG)


- [ ] The Picture below is from your Design 

![fig2](https://user-images.githubusercontent.com/63972670/160004074-d29064d9-4534-4837-9f4c-fbbb8fc68aa9.JPG)